Firestore/core/src/model/patch_mutation.cc

```diff
@@ -113,7 +113,10 @@ PatchMutation::Rep::GetPatch() const {
   std::map<FieldPath, absl::optional<google_firestore_v1_Value>> result;
   for (const FieldPath& path : mask_) {
     if (!path.empty()) {
-      result[path] = value_.Get(path);
+      auto value = value_.Get(path);
+      result[path] =
```

Comment:
Small nit: would using `if/else` be a little cleaner? (the type can be omitted in that case)
```cpp
if (value) {
  result[path] = DeepClone(*value);
} else {
  result[path] = absl::nullopt;
}
```

Reply:
Looks much better. Thanks.